src/combinators/TWAP.sol

```diff
@@ -82,7 +87,8 @@ contract TWAP is Ward {
         } else {
             ttl += config.ttl;
         }
-        fb.push(tag, bytes32((nexttally - pseudotally) / config.range), ttl);
+
+        fb.push(config.ttag, bytes32((nexttally - pseudotally) / config.range), ttl);
```

Comment:
the Config struct has a source address, it should also have a source tag.  more useful to be able to change the sources of twap data than to change the destination.  also we might want multiple twap configs per source

Reply:
See [twap map config by dttag, set src tag in config](https://github.com/feedbase/feedbase/pull/97/commits/0fe30c4567a8cf3a9d1be8b743426c13301d88dc)